### DISCUSSION TITLE
SplitButton add tick font awesome icon

### DISCUSSION THREAD

**Discussion by vnomohan** [primefaces/community#523] [PrimeNG] [answered]
hi there,
how to add splitbutton <i class="pi pi-check"></i> icon in programatically when user clicked to that label.
like below : 
<img width="307" alt="Screenshot 2023-11-22 at 9 54 14 AM" src="https://github.com/primefaces/community/assets/45278418/90b05292-01ea-49eb-9cd0-999618979b64">

I have tried with queryselector by appendchild element.. it is chaging via console browser  but then immediately it reverting back to original html content. why? please advise.

**Comment by vnomohan** [comment]
anyone?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

The approach you're using is not advised. You can use `icon` or `item` templates to do that.

**Comment by vnomohan** [comment]
hi ceti,
possible to share me the code or any references pls?

**Reply by vnomohan** [reply]
PrimeNG  is awesome and useful plugin but sad to say the support provided is not as expected...till now did not get proper reply though...
